flow-typed/npm/react-native-progress_vx.x.x.js

```diff
@@ -0,0 +1,87 @@
+// flow-typed signature: cda95545105831ec898fa23792e07ae9
+// flow-typed version: <<STUB>>/react-native-progress_v4.1.2/flow_v0.128.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'react-native-progress'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'react-native-progress' {
+  declare module.exports: any;
+}
+
+/**
+ * We include stubs for each file inside this npm package in case you need to
+ * require those files directly. Feel free to delete any files that aren't
```

Comment:
Please do delete all the files you don't use here.

Reply:
its the stubs required to tackle with flow warning as suggested in [CZO thread](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/flow.20types.20on.20external.20libs), well I removed some library from here that are not in use.